CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
 * Enable asynchronous tile uploader by default to improve animation performance. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
 * Vector tiles without symbols are not hold for fade-out animation so that less amount of vector tiles are managed at a time. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Migrated telemetry APIs from [mobile-events-android](https://github.com/mapbox/mapbox-events-android) to common SDK implementation. ([1672](https://github.com/mapbox/mapbox-maps-android/pull/1672))
```

Comment:
Should we drop a note about known issues with bearing = 0 ?

Reply:
I don't think we ever linked known issues to the change log, we will patch the issue shortly in beta.2 or rc release.